metal-perf/int8mm.mm

```diff
@@ -298,10 +308,12 @@ Int8MMMat4OpDescriptor reduce_mat4_int8mm(device, "reduce_mat4_int8mm", M, N,
                                           K);
     Int8MMBlockOpDesciptor reduce_group_int8mm(device, "reduce_group_int8mm", M,
                                                N, K);
-    reduce_vec4_int8mm.benchmark<BFloat16>();
-    reduce_mat4_int8mm.benchmark<BFloat16>();
-    reduce_group_int8mm.benchmark<BFloat16>();
-    naive_int8mm.benchmark<BFloat16>();
+    Int8MMLlamaCppDescriptor llama_cpp_int8mm(device, "llama_cpp_int8mm", M, N, K);
+    reduce_vec4_int8mm.benchmark<float>();
+    reduce_mat4_int8mm.benchmark<float>();
+    reduce_group_int8mm.benchmark<float>(1e-3, 1e-2);
+    naive_int8mm.benchmark<float>();
+    llama_cpp_int8mm.benchmark<float>(2e-3, 2e-2);
```

Comment:
Just curious, why changing those to float. Also, you'll need to change elemntSize to 4 (otherwise this code does a lot of out-of-bounds writes)

Reply:
Major reason is my macbook doesn't support bfloat. I can try `half` though, to make sure I wrote the template correctly.